HHS/Head-Start-TTADP
# [PROD] Warn of simultaneous editing of reports

## Description of change
This release includes
- A feature where users editing a report simultaneously will see a warning message
- The page title for the goals and objectives tab was set to something more meaningful to make tracking in Google Analytics easier
- Internal developer documentation was updated
- A reason was added to the modal that pops up when a user closes a goal from the recipient record
- An additional integration test was added

## How to test
- If you have tandem testing abilities, you and a partner can edit the same report on staging and see the warning message. 
- To test the page title for the goals and objectives tab from the RTR, simply visit that page and note that the title is distinct (includes the recipient's name)
- From that same page, close a goal and observe the "Regional office request" message is present

The other changes are either on the CI pipeline or updates to documentation and are tested implicitly by successful deploys.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-251
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1289
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1322


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
